src/Domain/Entity/Episode.php

```diff
@@ -35,6 +35,7 @@ public function __construct(
         int $aggregatedBroadcastsCount,
         int $availableClipsCount,
         int $aggregatedGalleriesCount,
+        bool $isEmbeddable,
```

Comment:
What does this mean?

Reply:
Some clips can be embedded into other sites, this is controlled by a field in PIPs, which we need to expose in the domain object for the frontend PR to have the info

Reply:
Should this have a docblock? Or be changed to `$isExternallyEmeddable` or something?

Reply:
there's about 100 parameters here, so a docblock may not be that helpful, but I can change the name if people think that's clearer